test/peliasDocGeneratorsTest.js

```diff
@@ -27,84 +27,94 @@ tape('create', function(test) {
         name: 'name 2',
         lat: 13.131313,
         lon: 31.313131,
-        place_type: 'macroregion',
+        place_type: 'dependency',
         bounding_box: '-13.691314,49.909613,1.771169,60.847887'
       },
       3: {
         id: 3,
         name: 'name 3',
         lat: 14.141414,
         lon: 41.414141,
-        place_type: 'region',
+        place_type: 'macroregion',
         bounding_box: '-13.691314,49.909613,1.771169,60.847888'
       },
       4: {
         id: 4,
         name: 'name 4',
         lat: 15.151515,
         lon: 51.515151,
-        place_type: 'macrocounty',
+        place_type: 'region',
         bounding_box: '-13.691314,49.909613,1.771169,60.847889'
       },
       5: {
         id: 5,
         name: 'name 5',
         lat: 16.161616,
         lon: 61.616161,
-        place_type: 'county',
+        place_type: 'macrocounty',
         bounding_box: '-13.691314,49.909613,1.771169,60.847890'
       },
       6: {
         id: 6,
         name: 'name 6',
         lat: 17.171717,
         lon: 71.717171,
-        place_type: 'localadmin',
+        place_type: 'county',
         bounding_box: '-13.691314,49.909613,1.771169,60.847891'
       },
       7: {
         id: 7,
         name: 'name 7',
         lat: 18.181818,
         lon: 81.818181,
-        place_type: 'borough',
+        place_type: 'localadmin',
         bounding_box: '-13.691314,49.909613,1.771169,60.847892'
       },
       8: {
         id: 8,
         name: 'name 8',
         lat: 19.191919,
         lon: 91.919191,
+        place_type: 'borough',
+        bounding_box: '-13.691314,49.909613,1.771169,60.847893'
+      },
+      9: {
+        id: 9,
+        name: 'name 9',
+        lat: 10.101010,
+        lon: 1.010101,
         place_type: 'locality',
-        bounding_box: '-13.691314,49.909613,1.771169,60.847893',
+        bounding_box: '-13.691314,49.909613,1.771169,60.847894',
```

Comment:
This test, as evidenced by the weird diff in the many lines above, is getting unweildy. Maybe we can split it up. Do all these cases really have to be tested in a stream?